.github/workflows/release.yml

```diff
@@ -29,4 +29,4 @@ jobs:
         env:
           MOBILE_MAVENCENTRAL_USER: ${{ secrets.MOBILE_MAVENCENTRAL_USER }}
           MOBILE_MAVENCENTRAL_PASSWORD: ${{ secrets.MOBILE_MAVENCENTRAL_PASSWORD }}
-        run: 'bash ./gradlew closeAndReleaseRepository'
+        run: 'bash ./gradlew closeAndReleaseSonatypeStagingRepository'
```

Comment:
In mistica we are using `closeAndReleaseStagingRepository`, I'm not sure if here it has to be the same.

I would check if we can remove the previous step which waits a minute, in mistica we don't have it and it works.

Reply:
Done!